app/src/main/java/xyz/whereuat/whereuat/ui/views/LatoEditTextView.java

```diff
@@ -0,0 +1,45 @@
+package xyz.whereuat.whereuat.ui.views;
```

Comment:
can we factor this function into a util or something? it's the exact same thing as the one in `LatoTextView`